ui/apps/platform/cypress/helpers/networkGraphPF.js

```diff
@@ -47,10 +47,8 @@ export function selectNamespace(namespace) {
         cy.get(networkGraphSelectors.selector.namespaceSelect).click();
         // Exact match to distinguish stackrox from stackrox-operator namespaces.
         cy.get(
-            `${selectSelectors.patternFlySelect.openMenu} .pf-c-select__menu-item [data-testid="namespace-name"]`
-        )
-            .contains(new RegExp(`^${namespace}$`))
```

Comment:
@vjwilson I removed this regex and made it an exact match instead. Seems like the regex wasn't working. Do we need to do this anyways? Seems like we should know exactly what we want to match anyways, no?